docs/src/api/class-keyboard.md

```diff
@@ -2,7 +2,7 @@
 * since: v1.8
 
 Keyboard provides an api for managing a virtual keyboard. The high level api is [`method: Keyboard.type`], which takes
-raw characters and generates proper keydown, keypress/input, and keyup events on your page.
+raw characters and generates proper `keydown`, `keypress`/`input`, and `keyup` events on your page.
```

Comment:
❓ Please let me know if you'd like to keep or revert this change. Happy to do either.